packages/worker/src/worker.ts

```diff
@@ -219,24 +222,56 @@ export class Worker {
           });
         }
       }
-      return new this(nativeWorker, workflowCreator, compiledOptions);
+      return new this(nativeWorker, workflowCreator, compiledOptions, dataConverter);
     } catch (err) {
       // Deregister our worker in case Worker creation (Webpack) failed
       await nativeWorker.completeShutdown();
       throw err;
     }
   }
 
+  public static async getDataConverter(options: WorkerOptions): Promise<DataConverter> {
```

Comment:
You're going to need this in the worker threads soon

Reply:
Here's the worker thread part. I haven't duplicated the error handling, since `getDataConverter` runs first

![image](https://user-images.githubusercontent.com/251288/146477576-997f6655-6253-4f53-8e48-fe0aeb51fd26.png)

![image](https://user-images.githubusercontent.com/251288/146477535-0373c5b7-75d5-49bb-a176-c0d26e8f0ee4.png)